covid_xprize/standard_predictor/xprize_predictor.py

```diff
@@ -300,30 +304,35 @@ def _create_country_samples(df: pd.DataFrame, geos: list) -> dict:
             cdf = cdf[cdf.ConfirmedCases.notnull()]
             context_data = np.array(cdf[context_column])
             action_data = np.array(cdf[action_columns])
-            outcome_data = np.array(cdf[outcome_column])
             context_samples = []
             action_samples = []
-            outcome_samples = []
-            nb_total_days = outcome_data.shape[0]
+            if is_training:
+                outcome_data = np.array(cdf[outcome_column])
+                outcome_samples = []
+                nb_total_days = context_data.shape[0]
+            else:
+                nb_total_days = context_data.shape[0] + 1
```

Comment:
When predicting, we do not need the outcomes (y) anymore, and we want to include the last day of context and action samples.